docs/index.md

```diff
@@ -54,6 +54,15 @@ Example applications are also provided in Python. For example,
 
     ./bin/spark-submit examples/src/main/python/pi.py 10
 
+Spark also provides a R API. To run Spark interactively in a R interpreter, use
```

Comment:
I think here (or somewhere close by) we should say that SparkR is an experimental component in `<SPARK_VERSION>` and that only the RDD API and DataFrame APIs have been implemented in SparkR.